resources/aws_ec2.go

```diff
@@ -154,7 +186,11 @@ type AwsEc2Res struct {
 	// UserData is used to run bash and cloud-init commands on first launch.
 	// See http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/user-data.html
 	// for documantation and examples.
-	UserData string `yaml:"userdata"`
+	UserData    string `yaml:"userdata"`
+	Bootstrap   bool   `yaml:"bootstrap"`   // install mgmt on the aws instance and phone home
```

Comment:
Can `Bootstrap` be a useful value instead of a bool? Just something to think about. Let's discuss here before you change it.

Reply:
I suppose it's a redundant field. If binary, remotegraph and seed are set, we know they want to bootstrap. Did you have something particular in mind?

Reply:
Not really, but I didn't think about it yet. Think on it, and let me know. I'll try and put myself in the mode and do the same.